<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1165</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="gbFactorial">
      <property name="title">
       <string>Faktoriál</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lFactorialValue">
         <property name="text">
          <string>Vstupní hodnota:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="leFactorialValue"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QProgressBar" name="pbFactorial">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblFactorialResult">
         <property name="text">
          <string>Výsledek:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>452</width>
            <height>423</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="lblFactorialResultValue">
             <property name="maximumSize">
              <size>
               <width>800</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="btnFactorialStart">
           <property name="text">
            <string>Zahájit výpočet</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnFactorialPauseOrResume">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Pozastavit výpočet</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnFactorialCancel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Zrušit výpočet</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gbEratosthenes">
      <property name="title">
       <string>Prvočísla až do</string>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="lErastothenesValue">
         <property name="text">
          <string>Vstupní hodnota:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="leErastothenesValue"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QProgressBar" name="pbEratosthenes">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblEratosthenesResult">
         <property name="text">
          <string>Výsledek:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>451</width>
            <height>423</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="lblEratosthenesResultValue">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="btnEratosthenesStart">
           <property name="text">
            <string>Zahájit hledání</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnEratosthenesPauseOrResume">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Pozastavit hledání</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnEratosthenesCancel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Zrušit hledání</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1165</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
